R/Subsets.R

```diff
@@ -453,6 +645,8 @@ CohortSubsetDefinition <- R6::R6Class(
     .subsets = list(),
     .subsetIds = c(),
     .targetOutcomePairs = c(),
+    .targetCohortWindow = Window$new(),
+    .subsetCohortWindow = Window$new(),
```

Comment:
`.targetCohortWindow` and `.subsetCohortWindow` are in the wrong class - these should be in the createCohortSubset